src/freenet/node/Version.java

```diff
@@ -50,7 +50,7 @@
 	public static final String protocolVersion = "1.0";
 
 	/** The build number of the current revision */
-	private static final int buildNumber = 1478;
+	private static final int buildNumber = 1480;
```

Comment:
Even if this might be correct it should only be changed by a merge when we actually make a new release, I guess…

Reply:
I moved it out now -- thank you again for the review! I created a cleaned up commit series at https://github.com/freenet/fred/pull/628